media-libs/aalib/files/aalib-1.4_rc6-key-down-OOB.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur a/src/aalinuxkbd.c b/src/aalinuxkbd.c
+--- a/src/aalinuxkbd.c	2001-04-26 16:37:31.000000000 +0200
++++ b/src/aalinuxkbd.c	2016-12-08 11:49:14.516042471 +0100
+@@ -165,10 +165,7 @@
+ 
+ static void blank_key_down(void)
+ {
+-    int f;
+-
+-    for (f = 0; f < NR_KEYS; f++)
+-	key_down[f] = 0;
++    memset(key_down, 0, NR_KEYS & 0xFF);
```

Comment:
Didn't you write that NR_KEYS is 256 ?
That'd reduce to '0x100 & 0xFF', which is 0, so basically making this function a no-op.